index.html

```diff
@@ -1,20 +1,207 @@
+                            alt="Facebook icon"></a></li>
+                <li class="d-inline"><a href="https://www.instagram.com/wix/"><img
+                            src="https://static.wixstatic.com/media/01c3aff52f2a4dffa526d7a9843d46ea.png/v1/fill/w_20,h_20,al_c,q_85,usm_0.66_1.00_0.01/01c3aff52f2a4dffa526d7a9843d46ea.webp"
+                            alt="Instagram icon"></a></li>
+                <li class="d-inline"><a href="https://soundcloud.com/"><img
+                            src="https://static.wixstatic.com/media/03ea2af965014a3eb32df628a4c18c0e.png/v1/fill/w_20,h_20,al_c,q_85,usm_0.66_1.00_0.01/03ea2af965014a3eb32df628a4c18c0e.webp"
+                            alt="SoundCloud icon"></a></li>
+            </ul>
+            <div class="col-1">
+                <a href=""><img src="img/menu-hamburger.svg" alt="hamburger menu"></a>
+            </div>
+        </div>
+
+
+
+
+
+
+
+
+
+    </header>
+
+    <main>
+        <section class="container-fluid hero-image">
+            <div class="row justify-content-center">
+                <div class="col-6 offset-3">
+                    <h1>DEW FESTIVAL MAY 28</h1>
+                    <h2>May 28, 2023, 9:00 AM</h2>
+
+                    <button class="ticket-button" type="button" aria-placeholder="Buy Tickets">Buy
+                        Tickets</button>
+                </div>
+        </section>
+        <section class="container-fluid line-up-section">
+            <div class="row">
+                <div class="col-10 offset-1 col-md-8 offset-md-2">
+                    <h3>LINEUP</h3>
+                    <p>ANDRE'S BEST / BLAKE ADAMSON / CAMMILLE / DJ CANOPY / DELORA PLANK / BELL HARRIS / NADINE BELMONT
+                        /
+
+                        DJ LACI GREY / DJ WILKIE / GYLES D /
+
+                        AXEL WRAY / ORSON BECKETT / DJ SHERRI / MUFFA LIVE / DJ SHORTCUT /
+
+                        LOVED UP SOUNDSYSTEM / BINGO T / AARON BAILEY / ABSTRACT MATERIAL LIVE / ADRIANA X
+                    </p>
+
+                </div>
+
+
+            </div>
+
+        </section>
+        <section class="section-three">
+
+        </section>
+        <section class=" container-fluid ticket-section">
+            <div class="row">
+                <div class="col-8 offset-2">
+                    <h3>DEW FESTIVAL 2023</h3>
+                    <h2>TICKETS ON SALE NOW</h2>
+                    <p>I'm a paragraph. Click here to add your own text and edit me.</p>
+                    <button class="ticket-button" type="button">Buy Tickets</button>
+                </div>
+
+            </div>
+
+
+        </section>
+
+        <section class="container-fluid about">
+            <div class="row">
+                <div class="col-10 offset-1">
+                    <h2>ABOUT</h2>
+                    <p>I'M A PARAGRAPH. CLICK HERE TO ADD YOUR OWN TEXT AND EDIT ME. IT'S EASY. JUST CLICK "EDIT TEXT"
```

Comment:
I notice you haven't set your own font-sizes to the headings. Notice in the design, the 'ABOUT' heading is actually smaller than the paragraph text.